lib/fastlane/plugin/revenuecat_internal/helper/revenuecat_internal_helper.rb

```diff
@@ -41,16 +41,16 @@ def self.auto_generate_changelog(repo_name, github_token, rate_limit_sleep)
         body = JSON.parse(resp[:body])
         commits = body["commits"].reverse
 
+        supported_types = ["breaking", "build", "ci", "docs", "feat", "fix", "perf", "refactor", "style", "test"]
```

Comment:
suuuuper minor, FFTI since it won't make any tangible difference in practice:
this _could_ be a set by adding `to_set` at the end, i.e.: 
```suggestion
        supported_types = ["breaking", "build", "ci", "docs", "feat", "fix", "perf", "refactor", "style", "test"].to_set
```

which would make the select on line `70` run _slightly_ faster.